expression/core/tagged_union.py

```diff
@@ -38,13 +38,23 @@ def tagged_union(
             items will be compared as the tuple (index, value)
         eq: If True, the __eq__ method will be generated.
     """
+    # Performance can be improved if it is a issue in another day
```

Comment:
What do you mean by this comment? Assigning two methods are not a performance issue. Is it the all of the transform function you are talking about? Please explain.

Reply:
@denghz Can we remove this line from the PR? I'm unsure why you added it?